src/olympia/landfill/management/commands/generate_addons.py

```diff
@@ -38,9 +38,6 @@ class Command(BaseCommand):
     )
 
     def handle(self, *args, **kwargs):
-        if not settings.DEBUG:
-            raise CommandError('You can only run this command with your '
-                               'DEBUG setting set to True.')
```

Comment:
Eventually this should be reverted or at least re-add it, comment it out and add a fat `TODO` or so